slither/printers/summary/evm.py

```diff
@@ -0,0 +1,100 @@
+"""
+    Module printing evm mapping of the contract
+"""
+
+from slither.printers.abstract_printer import AbstractPrinter
+from slither.evm import SourceToEVM, load_evm_cfg_builder
+
+
+    
+class PrinterEVM(AbstractPrinter):
+
+    ARGUMENT = 'evm'
+    HELP = 'Print the evm instructions of nodes in functions'
+
+    WIKI = 'https://github.com/trailofbits/slither/wiki/Printer-documentation#evm'
+
+    
+    def output(self, _filename):
+        """
+            _filename is not used
+            Args:
+                _filename(string)
+        """
+
+        evm_info = self._extract_evm_info(self.slither)
+        
+        for contract in self.slither.contracts_derived:
+            print('Contract {}'.format(contract.name))
```

Comment:
Do not use print, create a string and call `self.info` at the end.

See the other printers architecture

Reply:
Ah, will do. I adopted this style from `slithir` printer which has `print`s (with empty `self.info(txt)`) but looks like that is a bug. Will fix that too.